### ISSUE TITLE
Publications page changes

### ISSUE THREAD

**Issue by rachit2403** [KordingLab/KordingLab.github.io#38]
- [ ] I was thinking its better to add the publications from the last 3 years i.e (2019, 2018, 2017) and categorize as most recent publications. Anyway there is a link to Konrad's scholar page. 
- [ ] Each year will consist for all publications by anyone in the lab. 
- [ ] Attach a link to the code like previous articles.

**Comment by titipata** [comment]
I disagree with the first one. We should just update all of the publications. Our page is typically a place to host PDF files that allow anyone to access.

**Comment by aribenjamin** [comment]
Any additional work to be done here? 

What if we put a tSNE scatter plot of papers, with a mouseover that shows the title? That'd be awesome

**Comment by titipata** [comment]
Sounds great @aribenjamin. Bokeh is a good option here. However, we kinda want to point the HTML of the plot back to publication page, not putting the full HTML to the `publications.md`
